os/pm/pm_metrics.h

```diff
@@ -22,13 +22,23 @@
 #include <time.h>
 #include <queue.h>
 #include "pm.h"
+#include <tinyara/pm/pm.h>
 
 struct pm_statechange_s {
 	sq_entry_t entry;
+	sq_queue_t accum;
```

Comment:
This variable can be confused, because of the member variable of pm_accumchange_s.
Please rename this.